google/cloud/bigtable/row.py

```diff
@@ -281,6 +281,21 @@ def __ne__(self, other) -> bool:
         return not self == other
 
 
+class _LastScannedRow(Row):
```

Comment:
Upstream layers will update the `last_seen_key` value for all Rows, but filter out `_LastScannedRow` Rows rather than yield them to the user.

I wouldn't say it has to be a subclass, but this makes it convenient to grab the row_key consistently for all outputs